aws-codepipeline-transition-disabler/.gitignore

```diff
@@ -0,0 +1,5 @@
+/target/
```

Comment:
I've added one for `/src/test/java` but my local version doesn't have either resources directory and it builds fine, so I'm thinking those are fine without.